src/web/src/services/YacsService.js

```diff
@@ -36,8 +36,8 @@ const _getCourseIdentifier = courseObj => {
  * Returns a list of all courses
  * @returns {Promise<Course[]>}
  */
-export const getCourses = () =>
-  client.get('/class').then(({ data }) => {
+export const getCourses = (semester) =>
+  client.get(`/class?semester=${semester}`).then(({ data }) => {
```

Comment:
nitpick but can provide url paramters to `axios` through url parameters, think its like `get(..., {params: {semester})` or something

Reply:
ok, will do